.gitignore

```diff
@@ -1,5 +1,8 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
+# package-lock
+package-lock.json
```

Comment:
We're using yarn for this project so we shouldn't be seeing `package-lock.json` anyway. If it's in your directory, just delete it.